solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -127,7 +137,7 @@ describe("RandomBeacon - Group Creation", () => {
       })
 
       it("should lock the sortition pool", async () => {
-        expect(await sortitionPool.isLocked()).to.be.true
+        await expect(await sortitionPool.isLocked()).to.be.true
```

Comment:
we can remove "await" from `await expect(...)`

Reply:
6e5ced1cc2d78054a88226d3ea45fd8f605c2bd1